Recipely/Recipely/NetworkService/RecipeResponceDTO.swift

```diff
@@ -0,0 +1,30 @@
+// RecipeResponceDTO.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// RecipeResponseDTO
```

Comment:
```suggestion
/// RecipeResponseDTO
```
дока не соответствует действительности